### DISCUSSION TITLE
How to test local development env on actual mobile machine?

### DISCUSSION THREAD

**Discussion by mikinovation** [nuxt/nuxt#22804] [Questions] [answered]
How to test local development env on actual mobile machine?
I want to test local env on smartphone with nuxt3?

I did it with nuxt2 and cross-env before
```
cross-env NUXT_HOST=0.0.0.0 NUXT_PORT=3000 nuxt
```

you could test it when the PC and Smartphone were connected with same wifi

**Comment by onurusluca** [comment, marked as answer]
Have you tried `(p)npm dev -host`?
![image](https://github.com/nuxt/nuxt/assets/51825926/d2e316df-7d4f-417b-8440-d807b0ad20ef)

**Reply by mikinovation** [reply]
Thank you! I resolved it by my self...
I use WSL. However, It works when I setup on Windows
